src/Microsoft.Health.Fhir.Api/Features/Security/Authorization/ResourceActionHandler.cs

```diff
@@ -38,6 +55,17 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
             else
             {
                 context.Fail();
+
+                IFhirRequestContext fhirRequestContext = _fhirRequestContextAccessor.FhirRequestContext;
+
+                _auditLogger.LogAudit(
```

Comment:
Is this also logged by the middleware?

Reply:
I didn't see this code. Remove the logging from here and only logging in the middleware.